shared/common-adapters/text.desktop.js

```diff
@@ -50,8 +50,12 @@ class Text extends Component<Props> {
     return classNames.join(' ') || undefined
   }
 
-  _urlClick = () => {
-    this.props.onClickURL && openURL(this.props.onClickURL)
+  _urlClick = (e: MouseEvent) => {
+    if (!this.props.onClickURL) {
```

Comment:
without this, clicking on a URL that's a checkbox label will toggle the checkbox and open the URL